R/layers.R

```diff
@@ -206,8 +206,17 @@ write_file_attachments <- function(file_attachments, output_path) {
   output_path <- normalizePath(output_path, mustWork = TRUE)
 
   mapply(function(dest, src) {
-    if (fs::is_dir(src)) {
-      fs::dir_copy(src, file.path(output_path, dest), overwrite = TRUE)
+    if (dir.exists(src)) {
+      dest <- file.path(output_path, dest)
+      if (!dir.exists(dest)) {
+        dir.create(dest)
```

Comment:
Should this return value also be checked?

Reply:
I figured that if this fails, the file copy would fail and so the user would see an error there. But I can add a `dir.create2` that throws.